pkg/kubectl/cmd/drain.go

```diff
@@ -184,6 +185,7 @@ func NewCmdDrain(f cmdutil.Factory, out, errOut io.Writer) *cobra.Command {
 		},
 	}
 	cmd.Flags().BoolVar(&options.Force, "force", false, "Continue even if there are pods not managed by a ReplicationController, ReplicaSet, Job, DaemonSet or StatefulSet.")
+	cmd.Flags().BoolVar(&options.IgnoreNotFound, "ignore-not-found", false, "Ignore \"resource not found\" error when no matching controller is found.")
```

Comment:
The most reasonable seems to ignore pods managed by DaemonSets, if such a flag is set. Looking once again at the code, I think I should just re-use `--force` flag in `getPodCreator`, this way what I said above will make sense and will follow what's going on in the other parts of the code. I'll update the PR accordingly.